aptible/endpoints.go

```diff
@@ -0,0 +1,130 @@
+package aptible
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/reggregory/go-deploy/client/operations"
+	"github.com/reggregory/go-deploy/models"
+)
+
+// CreateEndpoint() creates Vhost API object + provision operation on the app.
+func (c *Client) CreateEndpoint(app_id int64, attrs map[string]interface{}) (*models.InlineResponse2019, error) {
+	service_id, err := c.GetServiceID(app_id)
+	if err != nil {
+		return nil, err
+	}
+
+	// Create Vhost API object
+	req_type := "http_proxy_protocol"
```

Comment:
I mention this on the terraform provider PR, but I think we'll want to allow this to be varied via terraform even if for now we just focus on a default value from there.